examples/cloudfunctions-stackdriver-intro/main.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
+"""Google Cloud Function for migrating a Google Compute Engine virtual machine.
```

Comment:
nit: use `#` comments (instead of multiline string" for license header please for consistency.